internal/app/tfsec/rules/azure/storage/container_delete_retention_policy_rule.go

```diff
@@ -0,0 +1,296 @@
+package storage
+
+// ATTENTION!
+// This rule was autogenerated!
+// Before making changes, consider updating the generator.
+
+// generator-locked
+import (
+	"github.com/aquasecurity/tfsec/internal/app/tfsec/block"
+	"github.com/aquasecurity/tfsec/internal/app/tfsec/scanner"
+	"github.com/aquasecurity/tfsec/pkg/provider"
+	"github.com/aquasecurity/tfsec/pkg/result"
+	"github.com/aquasecurity/tfsec/pkg/rule"
+	"github.com/aquasecurity/tfsec/pkg/severity"
+)
+
+func init() {
+	scanner.RegisterCheckRule(rule.Rule{
+		Provider:  provider.AzureProvider,
+		Service:   "storage",
+		ShortCode: "container_delete_retention_policy",
+		Documentation: rule.RuleDocumentation{
+			Summary:     "Container soft delete protects your data from being accidentally deleted by maintaining the deleted data in the system for a specified period of time. During the retention period, you can restore a soft-deleted container and its contents to the container's state at the time it was deleted. After the retention period has expired, the container and its contents are permanently deleted.",
```

Comment:
We generally use the summary to summarise the issue in basic terms.

The explanation goes into more detail about why the issue is a problem.

Reply:
Fixed the summary part.